### PR TITLE
Branch Deployment

### PR DESCRIPTION
**Pull request by erksch**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,10 +30,18 @@ jobs:
...
       - run: docker push bp2018hg1/tobito:$CIRCLE_WORKFLOW_ID
+      - run: docker push bp2018hg1/tobito:$CIRCLE_BRANCH
       - run: docker push bp2018hg1/tobito:latest
-  deploy:
+  deploy-staging:
+    docker:
+      - image: dtzar/helm-kubectl:2.11.0
+    steps:
+      - run: echo $CIRCLE_WORKING_DIRECTORY
+      - run: ls
```
Comment (Askir) [inline .circleci/config.yml]: ls? and echo here?

**Comment by Askir** [comment]
Oh another thing:
Apparently cert-manager doesn't support digitalocean as dns provider yet: https://github.com/jetstack/cert-manager/issues/1103
Can we change the url to something like tobito.askir.me/branch/branch-name? Shouldn't be THAT difficult to build with Ingress right? @erksch 
Otherwise I can try creating a wildcard cert manually that we will have to update manually as well.

File: .circleci/branch-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+kind: Ingress
+metadata:
+  name: tobito-${CIRCLE_BRANCH}
+  labels:
+    app: tobito-${CIRCLE_BRANCH}
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "ingress-ip"
+spec:
+  rules:
+  - host: ${CIRCLE_BRANCH}.tobito.aksir.me
```
Comment (Askir) [inline .circleci/branch-deployment.yaml]: typo here btw askir.me not aksir

Comment (Askir) [reply .circleci/branch-deployment.yaml]: And as I said we could probably change this to `tobito.askir.me/branch/${CURCLE_BRANCH}` right?

File: .circleci/config.yml (hunk 1 above)
Comment (erksch) [reply .circleci/config.yml]: I call it "echo ls"-Debugging

File: .circleci/branch-deployment.yaml (hunk 2 above)
Comment (erksch) [reply .circleci/branch-deployment.yaml]: I'll use /_branch/${CIRCLE_BRANCH}

**Pull request merged**